website/docs/d/data_factory_trigger_schedule.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Data Factory"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_data_factory_trigger_schedule"
+description: |-
+  Gets information about a trigger schedule in Azure Data Factory (Version 2).
```

Comment:
What do you mean by Version 2?

Reply:
Good question, it was all over the other ADF resources docs, so I just copied it over. Will remove.